clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -172,7 +173,9 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
         if (description.inService == null) {
           description.inService = sourceJob.inService
         }
-        description.migrationPolicy = description.migrationPolicy ?: sourceJob.migrationPolicy
+        if(description.disruptionBudget == null) {
```

Comment:
space before the `(` and perhaps an inline comment saying that the `migrationPolicy` should only be used when the `disruptionBudget` has not been specified.

Reply:
what if both are specified?

Reply:
Should the `disruptionBudget` be propagated from the `sourceJob`? (is that a thing?)

Reply:
good question , let me circle back on this with titus

Reply:
i think that will need some more work and most likely changes to the caching agents to, i will take a look at this next in a separate PR , this should fix the immediate issue i think.